src/utils/utils.ts

```diff
@@ -45,7 +45,7 @@ export class Utils {
                 { placeHolder: prompt, ignoreFocusOut: true });
         } else {
             if (item.prompt.responseType === 'bool') {
-                const oneProp = await vscode.window.showQuickPick(['True', 'False'],
+                const oneProp = await vscode.window.showQuickPick(['Yes (True)', 'No (False)'],
```

Comment:
You need to change the following condition check to pass user's yes input.

Reply:
Credit goes to @jrichter1

Reply:
Adding some tests, that method is not tested at all.